qiskit_experiments/curve_analysis/composite_curve_analysis.py

```diff
@@ -252,13 +252,16 @@ def _default_options(cls) -> Options:
                 the analysis result.
             plot (bool): Set ``True`` to create figure for fit result.
                 This is ``True`` by default.
+            return_fit_parameters (bool): (Deprecated) Set ``True`` to return all fit model parameters
+                with details of the fit outcome. Default to ``False``.
             extra (Dict[str, Any]): A dictionary that is appended to all database entries
                 as extra information.
         """
         options = super()._default_options()
         options.update_options(
             plotter=CurvePlotter(MplDrawer()),
             plot=True,
+            return_fit_parameters=False,
```

Comment:
```suggestion
            return_fit_parameters=True,
```
I think this has been True because options value comes from the base class. For example Ham tomo experiment has returned the fit parameter (in the unittest we just don't search by index)

Reply:
Hmm I ran ham tomo on current code but didn't see the fit parameter: 
![image](https://github.com/Qiskit-Extensions/qiskit-experiments/assets/3870315/6595cd35-a33a-43a6-8ce7-f8c8bf352a09)
You think we should change this behavior? I don't remember why `return_fit_parameters` was set to false for composite curve analysis.

Reply:
I realized we unintentionally introduced breaking API change in the previous release. 
https://github.com/Qiskit-Extensions/qiskit-experiments/blob/bc0b89e0eeedfabab7de2d5b6c5472e26bdbfb5a/qiskit_experiments/curve_analysis/composite_curve_analysis.py#L373-L375
We should have duplicated this `combined_summary` in the analysis results, but we only added this to artifact of the composite curve analysis. So the data has been moved to artifact without any warning. So far we didn't see any report in our issue, and I think this bug is not enough serious to motivate us to make a patch release for 0.6. And we don't need to redefine `return_fit_parameters` in the composite curve analysis options. What do you think?

Reply:
Ah, thanks for finding the source of the problem. How about I just keep the code the same and add a note in the Known Issues section of the release notes?

Reply:
Sounds good to me!